yarn.lock

```diff
@@ -0,0 +1,10827 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This project already has a package.lock file, it would be good to remove this yarn file to avoid conflicts.

Reply:
Yarn.lock was removed, you can see this change here - a3adb7711b1c365666e39e441dc24a1271c4fb87